DirectEmployers/MyJobs
# [PD-3676] refactor breadcrumb component

## What's this PR do?
PR refactors the breadcrumb component to be functional instead of class

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any analytics app or view the code

## How should this be manually tested?
View the code OR pull down repository. Go into any analytics app and make sure the breadcrumbs are still working

## Any background context you want to provide?
